client/header/activity-panel/panels/inbox/card.js

```diff
@@ -39,9 +39,116 @@ class InboxNoteCard extends Component {
 		}
 	}
 
+	openDismissModal( type ) {
+		this.setState( {
+			isDismissModalOpen: true,
+			dismissType: type,
+		} );
+	}
+
+	closeDismissModal() {
+		this.setState( {
+			isDismissModalOpen: false,
+		} );
+	}
+
+	renderDismissButton() {
+		return (
+			<Dropdown
+				position="bottom right"
+				renderToggle={ ( { onToggle } ) => (
+					<Button isTertiary onClick={ onToggle }>
+						{ __( 'Dismiss', 'woocommerce-admin' ) }
+					</Button>
+				) }
+				focusOnMount={ false }
+				popoverProps={ { noArrow: true } }
+				renderContent={ () => (
+					<ul>
+						<li>
+							<Button
+								onClick={ () =>
+									this.openDismissModal( 'this' )
+								}
+							>
+								{ __(
+									'Dismiss this message',
+									'woocommerce-admin'
+								) }
+							</Button>
+						</li>
+						<li>
+							<Button
+								onClick={ () => this.openDismissModal( 'all' ) }
+							>
+								{ __(
+									'Dismiss all message',
```

Comment:
Should this be 'Dismiss all messages'?

Reply:
Thank you @becdetat, you're totally right. I corrected it in the commit [cd71f7a](https://github.com/woocommerce/woocommerce-admin/pull/4262/commits/cd71f7abc8af42899e0e8d407b2af8960fefdc7b) and cherry-picked from the branch [add/4099_add_layout_changes](https://github.com/woocommerce/woocommerce-admin/pull/4256).